### ISSUE TITLE
define linting config

### ISSUE THREAD

**Issue by aaronhans** [cagov/design-system#73]
We are integrating code linting into the component development process. 

- Which approach should we use
- Which config should we use

Teams can get stuck on this step because it is a deep subject and people may have strong opinions about specific rules. In order to prevent a quagmire of discussions on each rule let's try to leverage some community work that has a lot of support.

There are a couple options we can review:
- The semistandard eslint configuration: https://www.npmjs.com/package/semistandard
- The open-wc eslint config: https://open-wc.org/guides/tools/linting-and-formatting/

**Comment by aaronhans** [comment]
Here are a couple PRs which let us compare the changes made when conforming to these specifications on the same component:

- The open-wc config: https://github.com/cagov/design-system/pull/70 (46 issues, 41 auto fixable)
- vs semistandard: https://github.com/cagov/design-system/pull/71 (28 issues, 28 auto fixable)

The open-wc version was stricter, seemed to cover everything semistandard did. In this case I found all the additional items it flagged were beneficial. I didn't run into anything annoying I felt was forcing me to rewrite things unnecessarily once I looked at all the things I had to fix manually.
